includes/admin/class-wcs-admin-meta-boxes.php

```diff
@@ -112,36 +113,44 @@ public function remove_meta_box_save( $post_id, $post ) {
 	 */
 	public function enqueue_styles_scripts() {
 		global $post;
+		$ver = WC_Subscriptions_Core_Plugin::instance()->get_plugin_version();
```

Comment:
This allows browser-cached scripts to be refreshed when this version number is updated.